sonic-syseepromd/scripts/syseepromd

```diff
@@ -144,6 +204,11 @@ class DaemonSyseeprom(daemon_base.DaemonBase):
             rcs = self.post_eeprom_to_db()
             if rcs != ERR_NONE:
                 self.log_error("Failed to post EEPROM to database")
+        try:
+            self.post_cpu_stats_to_db()
```

Comment:
Why do we choose syseepromd to do the job? It looks a litter bit weird because cpu/mem/disk info are not part of EEPROM.

Reply:
it is a daemon that takes statistics and we wanted to expand it to collect system info as well